metrics/plugins/prometheus/README.md

```diff
@@ -26,3 +26,18 @@ local httpd = require('http.server').new(...)
 ...
 httpd:route( { path = '/metrics' }, prometheus.collect_http)
 ```
+
+### Settings
+
+An example:
+```lua
+metrics = require('metrics')
+metrics.enable_default_metrics()
+
+prometheus = require('metrics.plugins.prometheus')
+metrics = require('http.server').new('0.0.0.0', 8080)
```

Comment:
It's better to use "httpd" or "http_server" here. It clashes with the first line.

Reply:
I mean
```lua
httpd = require('http.server').new('0.0.0.0', 8080)
router = require('http.router').new({charset = "utf8"})
httpd:set_router(router)
router:route( { path = '/metrics' }, prometheus.collect_http)
httpd:start()
```